services/storehost/spaceMon.go

```diff
@@ -31,20 +31,30 @@ import (
 	"go.uber.org/atomic"
 )
 
-const (
-	warnThreshold         = 200 * GiB
-	readOnlyThreshold     = 100 * GiB
-	resumeWritesThreshold = 200 * GiB
-)
+var (
+	// TODO: make these dynamically configurable!
+
+	// SpaceMonThresholdWarn if free-space below this, emit warning logs
+	SpaceMonThresholdWarn uint64 = 250 * GiB
 
-// monitoring interval
-const spaceMonInterval = 2 * time.Minute
+	// SpaceMonThresholdReadOnly if free-space below this, switch store to read-only
+	SpaceMonThresholdReadOnly uint64 = 200 * GiB
```

Comment:
@kirg Curious as to why do the thresholds need to be so high? Is this a uber use case or does cherami need the diskspace somehow?